Cannot find module 'graceful-fs'

I'm upgrading to 1.7.9 to fix #349 and get this error:
```
C:\Users\vubui\AppData\Local\.meteor\packages\meteor-tool\1.4.2_6\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:280
						throw(ex);
						^

Error: Cannot find module 'graceful-fs'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\vubui\AppData\Local\.meteor\packages\ostrio_files\1.7.9\npm\node_modules\fs-extra\lib\index.js:4:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
Exited with code: 1
Your application is crashing. Waiting for file change.
```

after upgrade meteor to 1.4.3.1 i got this error
`W20170215-01:24:19.718(8)? (STDERR) /Users/fengyibo/.meteor/packages/meteor-tool/.1.4.3_1.1r3yx81++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20170215-01:24:19.719(8)? (STDERR)                                             throw(ex);
W20170215-01:24:19.720(8)? (STDERR)                                             ^
W20170215-01:24:19.720(8)? (STDERR) 
W20170215-01:24:19.721(8)? (STDERR) Error: Cannot find module 'jsonfile'
W20170215-01:24:19.722(8)? (STDERR)     at Function.Module._resolveFilename (module.js:325:15)
W20170215-01:24:19.722(8)? (STDERR)     at Function.Module._load (module.js:276:25)
W20170215-01:24:19.722(8)? (STDERR)     at Module.require (module.js:353:17)
W20170215-01:24:19.722(8)? (STDERR)     at require (internal/module.js:12:17)
W20170215-01:24:19.723(8)? (STDERR)     at Object.<anonymous> (/Users/fengyibo/.meteor/packages/ostrio_files/.1.7.9.1nhyrtl++os+web.browser+web.cordova/npm/node_modules/fs-extra/lib/json/jsonfile.js:1:16)
W20170215-01:24:19.723(8)? (STDERR)     at Module._compile (module.js:409:26)
W20170215-01:24:19.734(8)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
W20170215-01:24:19.734(8)? (STDERR)     at Module.load (module.js:343:32)
W20170215-01:24:19.735(8)? (STDERR)     at Function.Module._load (module.js:300:12)
W20170215-01:24:19.735(8)? (STDERR)     at Module.require (module.js:353:17)
W20170215-01:24:19.735(8)? (STDERR)     at require (internal/module.js:12:17)
=> Exited with code: 1
`

--- a/.versions
+++ b/.versions
@@ -36,7 +36,7 @@ npm-mongo@1.5.45
 observe-sequence@1.0.12
 ordered-dict@1.0.8
 ostrio:cookies@2.1.3
-ostrio:files@1.7.9
+ostrio:files@1.7.10
 promise@0.8.3
 random@1.0.10
 reactive-var@1.0.10

--- a/files.coffee
+++ b/files.coffee
@@ -976,7 +976,12 @@ class FilesCollection
           chunkId: Match.Optional Number
         }
 
-        opts.binData = new Buffer(opts.binData, 'base64') if opts.binData
+        if opts.binData
+          if typeof Buffer.from == 'function'
+            opts.binData = Buffer.from opts.binData, 'base64'
+          else
+            opts.binData = new Buffer opts.binData, 'base64'
+
         _continueUpload = self._continueUpload opts.fileId
         unless _continueUpload
           throw new Meteor.Error 408, 'Can\'t continue upload, session expired. Start upload again.'
@@ -1916,7 +1921,7 @@ class FilesCollection
       opts =
         file:       @fileData
         fileId:     @fileId
-        chunkSize:  @config.chunkSize
+        chunkSize:  if @config.isBase64 then ((@config.chunkSize  / 4) * 3) else @config.chunkSize
         fileLength: @fileLength
       opts.FSName = @FSName if @FSName isnt @fileId
 

--- a/package.js
+++ b/package.js
@@ -1,6 +1,6 @@
 Package.describe({
   name: 'ostrio:files',
-  version: '1.7.9',
+  version: '1.7.10',
   summary: 'Upload files via DDP, HTTP and WebRTC/DC. To server FS, AWS, GridFS, DropBox or Google Drive.',
   git: 'https://github.com/VeliovGroup/Meteor-Files',
   documentation: 'README.md'

